config/environments/production.rb

```diff
@@ -37,13 +37,13 @@
 
   # Specifies the header that your server uses for sending files.
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
-  # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
+  config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.